web/themes/custom/hoeringsportal/templates/navigation/menu--secondary-navigation.html.twig

```diff
@@ -30,13 +30,13 @@
   {% import _self as menus %}
   {% if items %}
     {% if menu_level == 0 %}
-      <ul{{ attributes.addClass('navbar-nav is-secondary-navigation mr-auto float-right') }}>
+      <ul{{ attributes.addClass('navbar-nav is-secondary-navigation mr-lg-auto float-lg-right') }}>
     {% else %}
       <ul>
     {% endif %}
     {% for item in items %}
-      <li{{ item.attributes }} class="nav-item--secondary-navigation {{ item.in_active_trail ? 'active' : '' }}">
-        {{ link(item.title, item.url, { 'class':['nav-link', 'has-border']} ) }}
+      <li{{ item.attributes }} class="nav-item nav-item--secondary-navigation {{ item.in_active_trail ? 'active' : '' }}">
```

Comment:
Not directly related to the changes in this PR, but `{{ item.in_active_trail ? 'active' : '' }}` can be written as just `{{ item.in_active_trail ? 'active' }}` (cf. https://twig.symfony.com/doc/3.x/templates.html#other-operators).

Reply:
Thanks for the suggestion. It has been addressed in this commit: https://github.com/itk-dev/hoeringsportal/pull/67/commits/e74c6688664309f39336a0212894820868599eb5